README.rst

```diff
@@ -40,7 +40,15 @@ these files should be located at ``/edx/app/edxapp/`` directory::
 Location of Transformed Logs
 ****************************
 
-Transformed event logs can be found in ``/edx/var/logs/tracking/tracking.log`` file.
+Transformed events are logged using **'logging.handlers.SysLogHandler'** with **'facility: local2'**.
+
+To get logs, follow these steps:
+
+1. Create a log file with read/write permissions given to **syslog** user
+
+2. Create a mapping for **'local2'** in the configuration files present in **/etc/rsyslog.d/** ::
+
+    local2.*                 /path/of/file.log
```

Comment:
Shouldn't we specify a default path for the logging file?

Reply:
We can but I don't think we need to, any given path with the syslog permissions will work for this plugin.

Reply:
Yes, but we should recommend a default path anyways. `/edx/var/log/caliper/caliper.log` will suffice.

Reply:
ok.